### PR TITLE
Add Travis ci support

### PR DESCRIPTION
**Pull request by bugobliterator**
@pavel-kirienko I have also added the badge, which I think should work once you enable travis on this repo.

Sample Pass:
https://travis-ci.org/bugobliterator/dsdl/builds/243216412

Sample Fail:
https://travis-ci.org/bugobliterator/dsdl/builds/243206591

Resolves https://github.com/UAVCAN/dsdl/issues/18

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
```
Comment (pavel-kirienko) [inline .travis.yml]: Please replace this to 3.5, we're going to ditch the support for 2.7 in the future. Even now it is supported in a limited fashion.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+before_install:
+  - git clone https://github.com/UAVCAN/libuavcan
+  - pip install uavcan
+script:
+  - libuavcan/libuavcan/dsdl_compiler/libuavcan_dsdlc -v uavcan/
```
Comment (pavel-kirienko) [inline .travis.yml]: Perhaps we should avoid dependency on libuavcan and use PyUAVCAN directly? Here's the tutorial: http://uavcan.org/Implementations/Pyuavcan/Tutorials/4._Parsing_DSDL_definitions/. The necessary staements can be executed directly from the command line arguments. For example (I didn't test this):

```bash
python3 -c "from pyuavcan import dsdl; dsdl.parse_namespaces('uavcan')"
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 Standard DSDL definitions
 =========================
 
+[![Build Status](https://travis-ci.org/UAVCAN/dsdl.svg?branch=master)](https://travis-ci.org/UAVCAN/dsdl)
+
+
```
Comment (pavel-kirienko) [inline README.md]: Please remove the extra empty line.

**Comment by bugobliterator** [comment]
@pavel-kirienko Done!

**Comment by pavel-kirienko** [comment]
The Travis repo has been activated.

**Pull request merged**